README.md

```diff
@@ -22,11 +22,22 @@ The available configuration options are detailed below.
 |:--------|:-----------|
 | StatusHost | IP address or DNS of the NGINX+ instance to retrieve status information from. Defaults to `localhost`. |
 | StatusPort | Port the NGINX+ status endpoint can be reached at. Defaults to `8080`. |
+| APIType | API Type of the Nginx-plus(`legacy` or `newer`). Detects the type by Default. (see below) |
```

Comment:
Here, by this I mean, if the host machine has Nginx-plus version `R11` or `R12` then they should mention `legacy`, if the nginx-plus version is between `R16` to the `latest`(both inclusive) then they should mention `newer` and if `R13` to `R15` then as per the Nginx config they should mention `newer` or `legacy`.
So, here `legacy` or `newer` means the range of versions as mentioned above, not the specific version, therefore any version between `R11` to the `latest` is supported by our plugin. 
And by default, the plugin will detect the Nginx-plus version and determine whether it is a `legacy` or `newer` type of API is running on the host's machine.

Reply:
I believe I understand what you mean and how it's implemented.  My request is that instead of "newer" or "legacy" we have a "legacy" option and then a configurable newer version (1, 2, 3, etc.).  This provides more configurability and would better allow us to set a newer default of a tested version like 3 that works across a range of possible nginx plus releases.  As is, there could be a breaking change w/ version 8 and we'd blindly move to it without being able to recover without implementing the feature I'm asking for.

Reply:
Got your point, there was a misunderstanding from my side, sorry for that. This makes sense, I'll make the suggested changes.

Reply:
@rmfitzpatrick, I have made the suggested changes, please review the same.